src/app/contexts/ToggleContext/utils/constructTogglesEndpoint/index.js

```diff
@@ -1,6 +1,6 @@
 export default (service, origin) => {
   const requestOrigin = origin || 'https://www.test.bbc.com';
-  const baseTogglesUrl = `${process.env.SIMORGH_TOGGLES_URL}/toggles?application=simorgh&service=${service}&__amp_source_origin=${requestOrigin}`; // __amp_source_origin is relevant to both canonical and amp
+  const baseTogglesUrl = `${process.env.SIMORGH_TOGGLES_URL}?application=simorgh&service=${service}&__amp_source_origin=${requestOrigin}`; // __amp_source_origin is relevant to both canonical and amp
```

Comment:
Same here for consistency:
```suggestion
  const baseConfigUrl = `${process.env.SIMORGH_CONFIG_URL}?application=simorgh&service=${service}&__amp_source_origin=${requestOrigin}`; // __amp_source_origin is relevant to both canonical and amp
```